connectors/tests/test_sync.py

```diff
@@ -278,6 +278,17 @@ async def set_server_responses(
     mock_responses.post(
         f"{host}/.elastic-connectors-sync-jobs/_refresh", headers=headers, repeat=True
     )
+    mock_responses.post(
```

Comment:
I think that we should get rid of mock_responses at some point and mock the elasticsearch client directly and not the underlying aiohttp client.

Reply:
Indeed, but that will happen in the `Refactor SyncService class` PR. In this PR, I just want to make it green.